2015/03/20 05:32:08 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'ui002'' in D:\xampp\htdocs\cms\framework\db\CDbConnection.php:382
Stack trace:
#0 D:\xampp\htdocs\cms\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 D:\xampp\htdocs\cms\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 D:\xampp\htdocs\cms\framework\base\CModule.php(388): CDbConnection->init()
#3 D:\xampp\htdocs\cms\framework\base\CModule.php(104): CModule->getComponent('db')
#4 D:\xampp\htdocs\cms\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 D:\xampp\htdocs\cms\framework\web\CFormModel.php(40): CrudCode->init()
#6 D:\xampp\htdocs\cms\framework\gii\CCodeGenerator.php(152): CFormModel->__construct()
#7 D:\xampp\htdocs\cms\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('')
#13 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/bootstrap')
#14 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/gii/bootstrap
HTTP_REFERER=http://www.cms.com/gii
---
2015/03/20 05:42:01 [error] [exception.CException] exception 'CException' with message 'Alias "backdes.Attachment" is invalid. Make sure it points to an existing PHP file.' in D:\xampp\htdocs\cms\framework\YiiBase.php:318
Stack trace:
#0 D:\xampp\htdocs\cms\framework\gii\generators\crud\CrudCode.php(59): YiiBase::import('backdes.Attachm...', true)
#1 D:\xampp\htdocs\cms\framework\validators\CInlineValidator.php(43): CrudCode->validateModel('model', Array)
#2 D:\xampp\htdocs\cms\framework\validators\CValidator.php(197): CInlineValidator->validateAttribute(Object(BootstrapCode), 'model')
#3 D:\xampp\htdocs\cms\framework\base\CModel.php(158): CValidator->validate(Object(BootstrapCode), NULL)
#4 D:\xampp\htdocs\cms\framework\gii\CCodeGenerator.php(158): CModel->validate()
#5 D:\xampp\htdocs\cms\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#6 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#7 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#8 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#9 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('')
#11 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/bootstrap')
#12 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#13 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/gii/bootstrap
HTTP_REFERER=http://www.cms.com/gii/bootstrap
---
2015/03/20 05:58:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/
---
2015/03/20 05:58:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/
---
2015/03/20 05:58:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/
---
2015/03/20 05:58:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/
---
2015/03/20 05:58:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/
---
2015/03/20 05:58:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/
---
2015/03/20 05:58:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/
---
2015/03/20 05:58:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/
---
2015/03/20 05:58:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/styles.css".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('css/styles.css')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/css/styles.css
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 05:58:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site/index".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/index')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/site/index
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 05:58:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/styles.css".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('css/styles.css')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/css/styles.css
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 05:59:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/styles.css".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('css/styles.css')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/css/styles.css
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 05:59:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/styles.css".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('css/styles.css')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/css/styles.css
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 05:59:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/styles.css".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('css/styles.css')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/css/styles.css
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 05:59:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/styles.css".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('css/styles.css')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/css/styles.css
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 06:00:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/styles.css".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('css/styles.css')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/css/styles.css
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 06:00:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/styles.css".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('css/styles.css')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/css/styles.css
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 06:00:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/styles.css".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('css/styles.css')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/css/styles.css
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 06:00:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/styles.css".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('css/styles.css')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/css/styles.css
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 06:01:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/styles.css".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('css/styles.css')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/css/styles.css
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 06:04:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site/index".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/index')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/site/index
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 06:06:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site/login".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 06:07:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "content".' in D:\xampp\htdocs\cms\framework\web\CController.php:484
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(271): CController->missingAction('content')
#1 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('content')
#2 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('backdes/default...')
#3 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/backdes/default/content
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 06:07:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "content".' in D:\xampp\htdocs\cms\framework\web\CController.php:484
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(271): CController->missingAction('content')
#1 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('content')
#2 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('backdes/default...')
#3 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/backdes/default/content
HTTP_REFERER=http://www.cms.com/backdes
---
2015/03/20 06:18:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site/contact".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/contact')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/site/contact
HTTP_REFERER=http://www.cms.com/backdes/default
---
2015/03/20 06:18:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site/contact".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/contact')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/site/contact
HTTP_REFERER=http://www.cms.com/backdes/default
---
2015/03/20 06:18:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site/contact".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/contact')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/site/contact
---
2015/03/20 06:18:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site/contact".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/contact')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/site/contact
---
2015/03/20 06:18:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site/contact".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/contact')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/site/contact
---
2015/03/20 06:18:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site/contact".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/contact')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/site/contact
---
2015/03/20 06:18:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site/contact".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/contact')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/site/contact
---
2015/03/20 06:18:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site/contact".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/contact')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/site/contact
---
2015/03/20 06:18:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site/contact".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/contact')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/site/contact
---
2015/03/20 06:18:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site/contact".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/contact')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/site/contact
---
2015/03/20 06:19:50 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "contact".' in D:\xampp\htdocs\cms\framework\web\CController.php:879
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('contact', Array, true)
#1 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(60): CController->render('contact', Array)
#2 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionContact()
#3 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('contact')
#7 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/contact')
#8 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/site/contact
---
2015/03/20 06:20:48 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "login".' in D:\xampp\htdocs\cms\framework\web\CController.php:879
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('login', Array, true)
#1 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(83): CController->render('login', Array)
#2 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#3 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('login')
#7 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#8 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://www.cms.com/backdes/default
---
2015/03/20 06:22:11 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "login".' in D:\xampp\htdocs\cms\framework\web\CController.php:879
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('login', Array, true)
#1 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(83): CController->render('login', Array)
#2 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#3 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('login')
#7 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#8 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://www.cms.com/backdes/content?view=about
---
2015/03/20 06:22:59 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "login".' in D:\xampp\htdocs\cms\framework\web\CController.php:879
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('login', Array, true)
#1 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(83): CController->render('login', Array)
#2 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#3 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('login')
#7 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#8 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://www.cms.com/backdes/content?view=about
---
2015/03/20 06:23:00 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "login".' in D:\xampp\htdocs\cms\framework\web\CController.php:879
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('login', Array, true)
#1 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(83): CController->render('login', Array)
#2 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#3 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('login')
#7 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#8 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://www.cms.com/backdes/content?view=about
---
2015/03/20 06:23:00 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "login".' in D:\xampp\htdocs\cms\framework\web\CController.php:879
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('login', Array, true)
#1 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(83): CController->render('login', Array)
#2 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#3 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('login')
#7 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#8 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://www.cms.com/backdes/content?view=about
---
2015/03/20 06:23:00 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "login".' in D:\xampp\htdocs\cms\framework\web\CController.php:879
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('login', Array, true)
#1 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(83): CController->render('login', Array)
#2 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#3 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('login')
#7 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#8 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://www.cms.com/backdes/content?view=about
---
2015/03/20 06:23:00 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "login".' in D:\xampp\htdocs\cms\framework\web\CController.php:879
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('login', Array, true)
#1 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(83): CController->render('login', Array)
#2 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#3 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('login')
#7 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#8 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://www.cms.com/backdes/content?view=about
---
2015/03/20 06:23:01 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "login".' in D:\xampp\htdocs\cms\framework\web\CController.php:879
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('login', Array, true)
#1 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(83): CController->render('login', Array)
#2 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#3 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('login')
#7 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#8 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://www.cms.com/backdes/content?view=about
---
2015/03/20 06:23:01 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "login".' in D:\xampp\htdocs\cms\framework\web\CController.php:879
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('login', Array, true)
#1 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(83): CController->render('login', Array)
#2 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#3 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('login')
#7 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#8 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://www.cms.com/backdes/content?view=about
---
2015/03/20 06:23:02 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "login".' in D:\xampp\htdocs\cms\framework\web\CController.php:879
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('login', Array, true)
#1 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(83): CController->render('login', Array)
#2 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#3 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('login')
#7 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#8 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://www.cms.com/backdes/content?view=about
---
2015/03/20 06:23:02 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "login".' in D:\xampp\htdocs\cms\framework\web\CController.php:879
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('login', Array, true)
#1 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(83): CController->render('login', Array)
#2 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#3 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('login')
#7 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#8 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://www.cms.com/backdes/content?view=about
---
2015/03/20 06:24:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:28:58 [error] [php] include(Employee.php): failed to open stream: No such file or directory (D:\xampp\htdocs\cms\framework\YiiBase.php:418)
Stack trace:
#0 D:\xampp\htdocs\cms\protected\components\UserIdentity.php(20): spl_autoload_call()
#1 D:\xampp\htdocs\cms\protected\models\LoginForm.php(51): UserIdentity->authenticate()
#2 D:\xampp\htdocs\cms\framework\validators\CInlineValidator.php(43): LoginForm->authenticate()
#3 D:\xampp\htdocs\cms\framework\validators\CValidator.php(197): CInlineValidator->validateAttribute()
#4 D:\xampp\htdocs\cms\framework\base\CModel.php(158): CInlineValidator->validate()
#5 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(79): LoginForm->validate()
#6 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#7 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams()
#8 D:\xampp\htdocs\cms\framework\web\CController.php(287): SiteController->runAction()
#9 D:\xampp\htdocs\cms\framework\web\CController.php(266): SiteController->runActionWithFilters()
#10 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): SiteController->run()
#11 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController()
#12 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#13 D:\xampp\htdocs\cms\index.php(13): CWebApplication->run()
REQUEST_URI=/site/login
in D:\xampp\htdocs\cms\protected\components\UserIdentity.php (20)
in D:\xampp\htdocs\cms\protected\models\LoginForm.php (51)
in D:\xampp\htdocs\cms\protected\controllers\SiteController.php (79)
2015/03/20 06:30:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:35:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:35:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:35:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:35:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:35:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:35:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:35:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:36:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/page?view=about
---
2015/03/20 06:36:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/index
---
2015/03/20 06:36:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/contact
---
2015/03/20 06:36:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/page?view=about
---
2015/03/20 06:37:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:37:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:37:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:37:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:38:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:39:45 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.LoginForm" is not defined.' in D:\xampp\htdocs\cms\framework\base\CComponent.php:128
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\helpers\CHtml.php(2044): CComponent->__get('LoginForm')
#1 D:\xampp\htdocs\cms\framework\web\helpers\CHtml.php(1774): CHtml::resolveValue(Object(LoginForm), 'LoginForm')
#2 D:\xampp\htdocs\cms\framework\web\helpers\CHtml.php(1205): CHtml::activeInputField('text', Object(LoginForm), 'LoginForm', Array)
#3 D:\xampp\htdocs\cms\framework\web\widgets\CActiveForm.php(575): CHtml::activeTextField(Object(LoginForm), 'LoginForm', Array)
#4 D:\xampp\htdocs\cms\protected\views\site\login.php(25): CActiveForm->textField(Object(LoginForm), 'LoginForm')
#5 D:\xampp\htdocs\cms\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#6 D:\xampp\htdocs\cms\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#7 D:\xampp\htdocs\cms\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#8 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('login', Array, true)
#9 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(88): CController->render('login', Array)
#10 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#11 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#12 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#13 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('login')
#15 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#16 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#17 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/site/login
---
2015/03/20 06:39:51 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.LoginForm" is not defined.' in D:\xampp\htdocs\cms\framework\base\CComponent.php:128
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\helpers\CHtml.php(2044): CComponent->__get('LoginForm')
#1 D:\xampp\htdocs\cms\framework\web\helpers\CHtml.php(1774): CHtml::resolveValue(Object(LoginForm), 'LoginForm')
#2 D:\xampp\htdocs\cms\framework\web\helpers\CHtml.php(1205): CHtml::activeInputField('text', Object(LoginForm), 'LoginForm', Array)
#3 D:\xampp\htdocs\cms\framework\web\widgets\CActiveForm.php(575): CHtml::activeTextField(Object(LoginForm), 'LoginForm', Array)
#4 D:\xampp\htdocs\cms\protected\views\site\login.php(25): CActiveForm->textField(Object(LoginForm), 'LoginForm')
#5 D:\xampp\htdocs\cms\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#6 D:\xampp\htdocs\cms\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#7 D:\xampp\htdocs\cms\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#8 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('login', Array, true)
#9 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(88): CController->render('login', Array)
#10 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#11 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#12 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#13 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('login')
#15 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/login')
#16 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#17 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/site/login
---
2015/03/20 06:40:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:56:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/
---
2015/03/20 06:56:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 06:56:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/site/login
---
2015/03/20 07:34:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/
---
2015/03/20 07:35:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/
---
2015/03/20 07:35:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/modules/exporting.js".' in D:\xampp\htdocs\cms\framework\web\CWebApplication.php:280
Stack trace:
#0 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('js/modules/expo...')
#1 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/modules/exporting.js
HTTP_REFERER=http://www.cms.com/
---
2015/03/20 07:44:02 [error] [exception.CException] exception 'CException' with message 'Property "Content.title" is not defined.' in D:\xampp\htdocs\cms\framework\base\CComponent.php:128
Stack trace:
#0 D:\xampp\htdocs\cms\framework\db\ar\CActiveRecord.php(144): CComponent->__get('title')
#1 D:\xampp\htdocs\cms\protected\views\site\index.php(4): CActiveRecord->__get('title')
#2 D:\xampp\htdocs\cms\framework\web\CBaseController.php(127): require('D:\xampp\htdocs...')
#3 D:\xampp\htdocs\cms\framework\web\CBaseController.php(96): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#4 D:\xampp\htdocs\cms\framework\web\CController.php(870): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#5 D:\xampp\htdocs\cms\framework\web\CController.php(783): CController->renderPartial('index', Array, true)
#6 D:\xampp\htdocs\cms\protected\controllers\SiteController.php(31): CController->render('index', Array)
#7 D:\xampp\htdocs\cms\framework\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#8 D:\xampp\htdocs\cms\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#9 D:\xampp\htdocs\cms\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#10 D:\xampp\htdocs\cms\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(276): CController->run('index')
#12 D:\xampp\htdocs\cms\framework\web\CWebApplication.php(135): CWebApplication->runController('site/index')
#13 D:\xampp\htdocs\cms\framework\base\CApplication.php(162): CWebApplication->processRequest()
#14 D:\xampp\htdocs\cms\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/site/index
HTTP_REFERER=http://www.cms.com/site/page?view=about
---
